### ISSUE TITLE
CustomHBond force with 4 particles per donor/acceptor

### ISSUE THREAD

**Issue by cabb99** [openmm/openmm#2565]
Our group has been working on the implementation of the AWSEM forcefield and the 3SPN2 forcefield in openMM. During the execution of this project we found that some of the energy terms were better represented by using 4 particles per donor and aceptor, rather than 3. I wonder if it would be possible to add support for an extra particle both on the donor and aceptor to the CustomHBondForce.

**Comment by peastman** [comment]
Yes, I think that would be possible.  I'll mark this as an enhancement request.
